cmd/occollector/app/builder/processor_builder.go

```diff
@@ -78,6 +106,9 @@ type QueuedSpanProcessorCfg struct {
 	// SenderType indicates the type of sender to instantiate
 	SenderType   SenderType `mapstructure:"sender-type"`
 	SenderConfig interface{}
+	// BatchingConfig sets config parameters related to batching
+	BatchingConfig BatchingConfig `mapstructure:"batching"`
+	RawConfig      *viper.Viper
```

Comment:
Why this needs to be added?

Reply:
ah, it was needed for the followup which changes the other exporters to allow for batching. I must have missed its removal -- removed.